graql/reasoner/atom/binary/AttributeAtom.java

```diff
@@ -249,7 +251,7 @@ private boolean multiPredicateEqual(AttributeAtom that){
     @Override
     public final int hashCode() {
         int hashCode = this.alphaEquivalenceHashCode();
-        hashCode = hashCode * 37 + this.getVarName().hashCode();
+        hashCode = hashCode * 37 + this.getVarName().hashCode() + this.getAttributeVariable().hashCode();
```

Comment:
Could we follow the common hashing approach and make it something like:
```
hashCode = hashCode * 37 + this.getVarName().hashCode();
hashCode = hashCode * 37 + this.getAttributeVariable().hashCode();
```

Reply:
Ah I see, changed